src/test/java/Pages/WebCommands.java

```diff
@@ -18,19 +19,62 @@ public WebElement getElement(By locator) {
         return UseDriver.getDriver().findElement(locator);
     }
 
+
+    // scroll to TIME MACHINE button
+    public WebElement getTimeMachineElementByScroll(By locator) {
+        JavascriptExecutor js = (JavascriptExecutor) UseDriver.getDriver();     // Casting
+        js.executeScript("scrollBy(0,800);");
```

Comment:
There should not be any hard coded value (eg. 0,800) at WebCommands, rather it should be parameterized so that you can create custom method at page object level for scroll to any element.